src/nft/ZoraCreator1155Impl.sol

```diff
@@ -623,6 +698,25 @@ contract ZoraCreator1155Impl is
         }
     }
 
+    /// @notice Withdraws ETH from the Zora Rewards contract
+    function withdrawRewards(uint256 amount) public onlyAdminOrRole(CONTRACT_BASE_ID, PERMISSION_BIT_FUNDS_MANAGER) {
+        bytes memory data = abi.encodeWithSelector(IProtocolRewards.withdraw.selector, amount);
+
+        (bool success, ) = address(PROTOCOL_REWARDS).call(data);
+
+        if (!success) {
+            revert();
+        }
+
+        address fundsRecipient = config.fundsRecipient;
+
+        if (fundsRecipient != address(0)) {
```

Comment:
this should revert if funds recipient is address(0) - otherwise the funds will get locked in this contract